.changeset/dry-pillows-carry.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/core': minor
```

Comment:
Would say this is a patch change since it's just improving the existing behaviour without an API change